app/models/manageiq/providers/google/cloud_manager/provision_workflow.rb

```diff
@@ -0,0 +1,23 @@
+class ManageIQ::Providers::Google::CloudManager::ProvisionWorkflow < ::MiqProvisionCloudWorkflow
+  def allowed_instance_types(_options = {})
+    source = load_ar_obj(get_source_vm)
+    ems = get_targets_for_ems(source, :cloud_filter, Flavor, 'flavors')
+    ems.each_with_object({}) { |f, h| h[f.id] = display_name_for_name_description(f) }
+  end
+
+  def allowed_disk_sizes(_options = {})
+    min = 10
+    max = 10 * 1024 # 10 TB
+    (min..max).each_with_object({}) { |i, h| h[i] = i.to_s }
```

Comment:
Also, I assumed this was for validation. If we instead allowed the user to provide a number, would there be a way to check if it was in a specified range? 

GCE VMs allow for custom memory and CPU, which is when this might come in handy as well.